lua/telescope/builtin.lua

```diff
@@ -774,4 +774,33 @@ builtin.man_pages = function(opts)
   }):find()
 end
 
+builtin.colorscheme = function(opts)
```

Comment:
@glepnir That's very useful!!!! I wanted to create this function so bad! thank you ( also for Thinkvim). 

I wonder is it possible to include nvim-base16.lua as an option of its installed or loaded? and query from `base16.themes` array from selection, then on selection do `require'base16'(require'base16'.themes["selection"], true)` [reference](https://github.com/tami5/nvim/blob/main/lua/core/theme.lua#L75)

Reply:
Perhaps we can do `local colors = vim.list_extend(opts.colors or {}, vim.fn.getcompletion('', 'color'))` on line 780.

This would allow users to pass their own lists of colorschemes as well (if they are not currently available).

Reply:
yep.